ui/pages/first-time-flow/metametrics-opt-in/index.scss

```diff
@@ -51,8 +51,8 @@
   }
 
   &__content {
-    overflow-y: scroll;
```

Comment:
This property is redundant and is being overridden by `overflow-y: auto` below (which is being pulled up from the duplicate `&__content`).